astro.config.mjs

```diff
@@ -210,7 +212,11 @@ export default defineConfig({
               { label: 'Installation', link: '/dropins/checkout/installation/' },
               { label: 'Initialization', link: '/dropins/checkout/initialization/' },
               { label: 'Styles', link: '/dropins/checkout/styles/' },
-              { label: 'Containers',
+              // { label: 'Slots', link: '/dropins/checkout/slots/' },
```

Comment:
I think we need to normalize the order in which each section appears across components. Right now, there's variance.